source/release-notes.txt

```diff
@@ -10,6 +10,30 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.44.0
+----------------
+
+*Released September 3, 2024*
+
+New Features:
+
+- Support for users working with data stored on different connections 
+  stored in the Compass connections window. (:issue:`COMPASS-6410`).
```

Comment:
I think we should probably trim the period before the parenthesis, since we don't use them elsewhere.